days/day-00-example/test.sh

```diff
@@ -5,16 +5,16 @@ D=$(dirname $(realpath $0))
 
 echo ""
 echo "--- Day 0: Examples ---"
-$D/../../languages/rust.sh   $D/input.txt $D/output.txt $D/solutions/example.rs
-$D/../../languages/go.sh     $D/input.txt $D/output.txt $D/solutions/example.go
-$D/../../languages/c.sh      $D/input.txt $D/output.txt $D/solutions/example.c
-$D/../../languages/cpp.sh    $D/input.txt $D/output.txt $D/solutions/example.cpp
-$D/../../languages/sml.sh    $D/input.txt $D/output.txt $D/solutions/example.sml
-$D/../../languages/bash.sh   $D/input.txt $D/output.txt $D/solutions/example.bash
-$D/../../languages/php.sh    $D/input.txt $D/output.txt $D/solutions/example.php
-$D/../../languages/python.sh $D/input.txt $D/output.txt $D/solutions/example.py
-$D/../../languages/deno.sh   $D/input.txt $D/output.txt $D/solutions/example.ts
-$D/../../languages/node.sh   $D/input.txt $D/output.txt $D/solutions/example.mjs
-$D/../../languages/ruby.sh   $D/input.txt $D/output.txt $D/solutions/example.rb
-$D/../../languages/java.sh   $D/input.txt $D/output.txt $D/solutions Example
+$D/../../languages/rust.sh   $D/io/alice.* $D/solutions/example.rs
+$D/../../languages/go.sh     $D/io/alice.* $D/solutions/example.go
+$D/../../languages/c.sh      $D/io/alice.* $D/solutions/example.c
+$D/../../languages/cpp.sh    $D/io/alice.* $D/solutions/example.cpp
+$D/../../languages/sml.sh    $D/io/alice.* $D/solutions/example.sml
+$D/../../languages/bash.sh   $D/io/alice.* $D/solutions/example.bash
+$D/../../languages/php.sh    $D/io/alice.* $D/solutions/example.php
+$D/../../languages/python.sh $D/solutions/example.py $D/io/*
```

Comment:
:thinking: 

Should we just do the same globbing for solutions?
```suggestion
$D/../../languages/python.sh $D/solutions/*.py $D/io/*
```

Scope creep...  Ignore!
```suggestion
$D/../../languages/python.sh --solutions $D/solutions/*.py --io $D/io/*
```

Reply:
....although, this would actually enable us to have a `test.sh` at root-level, which runs all the tests for a single language, across all days, which would be pretty neat :thinking:

Reply:
**UPDATE** I actually found a trivial solution to the problem presented above. See cca54c1 for details.

Reply:
But now we are saying goodbye to sorting by performance...

Reply:
We might as well go as far as doing: `../../test-day.sh "solutions/*" "io/*"` at this point, and just don't sort at all.

Reply:
....unless someone has a trivial way to sort a list by a column in bash... :thinking:

Reply:
Resolving this conversation, by reverting e07e3bd, because it introduces complexity and indirection, which I am not comfortable to introduce at this point in time.